### ISSUE TITLE
Add more color controls

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brockgr/csshx#14]
```
Allow user to configure foreground and background colors for the master window 
"disabled" 
windows and the "enable" cursor window.

```

Original issue reported on code.google.com by `gavin.brock` on 9 Jul 2009 at 1:38

**Comment by GoogleCodeExporter** [comment]
```
Added better color support to SVN trunk version:

renamed color_disabled to color_disabled_foreground
added color_disabled_background

renamed color_selected to color_selected_background
added color_selected_foreground

If *_background is not used, transparency should be preserved.
```

Original comment by `gavin.brock` on 29 Jul 2009 at 3:48

**Comment by GoogleCodeExporter** [comment]
```
Added color_setbounds_* to SVN for completeness.
```

Original comment by `gavin.brock` on 30 Jul 2009 at 1:27
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released in 0.64: http://gavcode.wordpress.com/2010/03/30/csshx-0-64/

```

Original comment by `gavin.brock` on 30 Mar 2010 at 2:07
- Changed state: **Done**
